framework/src/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -2589,11 +2586,10 @@ private boolean checkMethodReferenceInference(
                 }
             }
             if (requiresInference) {
-                if (!typeArgumentInferenceCheck) {
+                if (checker.hasOption("conservativeUninferredTypeArguments")) {
```

Comment:
Can you update the test in `./framework/tests/all-systems/java8/memberref/MemberReferences.java` to make explicit why or why not to expect the error message.

Reply:
Done.